wiki/People/Beatmap_Nominators/en.md

```diff
@@ -29,6 +29,7 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
 ### What Qualifies as a Mod Post
 + Upon signing up you will be asked to provide the total amount of Kudosu as well as the amount of mod posts within the last 3 months, sorted by month upon signing up. So if the announcement is posted in April the mod posts for January, February and March will be relevant. The relevant months will also be listed on the submission form.
 + A mod post qualifies as one if it has received a Kudosu and has been created within the time span listed on the submission form.
++ Since November 2017 beatmaps affected under Modding V2 got a reward system implemented, based on upvotes/downvotes. The "mod post" equivalent in this case is a beatmap discussion, in which at least one suggestion/problem got awarded with an upvote.
 + The gamemode (osu!, osu!mania, osu!catch, osu!taiko) that you modded beatmaps in does not matter as your activity in the modding community as a whole is being measured and not the activity in a certain game mode.
```

Comment:
This is quite misleading. I chose to be less active in Mania after reading this and focused on fewer better mods, which was why I was rejected.